ts/components/session/LeftPaneSettingSection.tsx

```diff
@@ -212,6 +212,11 @@ export class LeftPaneSettingSection extends React.Component<Props, State> {
         title: window.i18n('privacySettingsTitle'),
         hidden: false,
       },
+      {
+        id: SessionSettingCategory.Blocked,
+        title: window.i18n('settingsUnblockHeader'),
+        hidden: isSecondaryDevice,
```

Comment:
this should be fine to be shown anytime, as we can currently unblock a user from a secondary device.
```suggestion
        hidden: false,
```